samples/client/petstore/cpp-qt5/client/PFXHelpers.h

```diff
@@ -50,7 +50,8 @@ class OptionalParam {
     }
 };
 
-bool setDateTimeFormat(const QString&);
+bool setDateTimeFormat(const QString &format);
```

Comment:
Is this a breaking change as there are changes in the method signature?

Reply:
same signature, just the parameter is not specified earlier and it is perfectly fine c++ thing

Reply:
yep, was just done for consistency within the file

Reply:
also can see that it was not a breaking change because this file did not need to be changed to compile https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/cpp-qt5/PetStore/main.cpp interesting enough, you can see that is trying to do a "similar" thing to this in a more roundabout way than just assigning IsoDataMs

Reply:
👌